recipes/povray/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "povray" %}
+{% set version = "3.7.0" %}
+{% set release = "8" %}
+{% set releaseversion = version + "." + release %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

Comment:
Why isn't the `8` in the version number? It looks like it should be `3.7.0.8`?

Reply:
I though conda is restricted to 3 digits

Reply:
My mistake, it seems to work just fine. Thanks again for your review.

Reply:
No worries, conda follows PEP 440 which allows any number of digits.